### DISCUSSION TITLE
In the preview, the overview field description has become shorter with the image.

### DISCUSSION THREAD

**Discussion by avinashtelsa** [Laravel-Backpack/community-forum#985] [Q&A (Help)] [answered]
In the first image, the icon isn't displayed, so the entire overview is shown. 
![first](https://github.com/Laravel-Backpack/community-forum/assets/166697230/177d9b43-f5bd-4095-bbec-d1ec5227d519)

But in the second image, I've displayed the icon, so the overview has become smaller.
![second](https://github.com/Laravel-Backpack/community-forum/assets/166697230/04e09bcd-95a3-4db7-946e-aacdea240fa6)

There, I want to display the entire overview just like in the first image. And here is my code.

protected function setupShowOperation()
    {
        //CRUD::setFromDb(); // set columns from db columns.
        CRUD::column('title');
        CRUD::addColumn([
            'name'=>'icon',
            'type'=>'image',
//            'disk'=>env('FILESYSTEM_DRIVER'),
//            'path' => 'uploads',
            //'prefix'=>'storage/',
        ]);
        CRUD::column('slug');
        CRUD::column('overview');
//        CRUD::addColumn([
//            'name' => 'overview',
//            'label' => 'Overview',
//            'class' => 'form-group col-md-4',
//        ]);
        CRUD::column('vimeo_link');
        CRUD::addColumn([
            'name' => 'hide_from_courses',
            'label' => 'Hide from courses',
            'type' => 'closure',
            'function' => function($entry) {
                return $entry->hide_from_courses ? 'Yes' : 'No';
            }
        ]);
        //CRUD::column('icon')->type('image');

        CRUD::column('created_at');
        CRUD::column('updated_at');
    }

**Comment by karandatwani92** [comment, marked as answer]
Hey @avinashtelsa

Try this `CRUD::column('overview')->type('text')->limit(300);`

**Reply by avinashtelsa** [reply]
Thank you, this is working properly.
